src/utils/format/formatDifficulty.ts

```diff
@@ -0,0 +1,16 @@
+const DIFFICULTIES = {
+  EASY: 'Fácil',
+  MEDIUM: 'Media',
```

Comment:
We should probably agree if we will called it Media o Medio. I used Medio for other components. I am fine with either of them